packages/@aws-cdk/aws-stepfunctions-tasks/lib/ecs/run-ecs-fargate-task.ts

```diff
@@ -4,6 +4,7 @@ import { CommonEcsRunTaskProps, EcsRunTaskBase } from './run-ecs-task-base';
 
 /**
  * Properties to define an ECS service
+ * @deprecated - replaced by `EcsRunTask` and `EcsRunTaskProps`
```

Comment:
```suggestion
 * @deprecated replaced by `EcsRunTask` and `EcsRunTaskProps`
```

(can you verify everywhere that there are no "-" prefixes)